### ISSUE TITLE
bogart failed with "Assertion `cnt > 0' failed"

### ISSUE THREAD

**Issue by SixPlusSeven** [marbl/canu#1320]
I am using canu v1.8 to assembly an genome with nanopore data, my job break off at bat step, here is my error log(unitigger.err):
```
==> PARAMETERS.

Resources:
  Memory                16 GB
  Compute Threads       16 (command line)

Lengths:
  Minimum read          0 bases
  Minimum overlap       500 bases

Overlap Error Rates:
  Graph                 0.144 (14.400%)
  Max                   0.144 (14.400%)

Deviations:
  Graph                 6.000
  Bubble                6.000
  Repeat                3.000

Edge Confusion:
  Absolute              2100
  Percent               200.0000

Unitig Construction:
  Minimum intersection  500 bases
  Maxiumum placements   2 positions

Debugging Enabled:
  (none)

==> LOADING AND FILTERING OVERLAPS.

ReadInfo()-- Using 2385317 reads, no minimum read length used.

OverlapCache()-- limited to 16384MB memory (user supplied).

OverlapCache()--      18MB for read data.
OverlapCache()--      90MB for best edges.
OverlapCache()--     236MB for tigs.
OverlapCache()--      63MB for tigs - read layouts.
OverlapCache()--      90MB for tigs - error profiles.
OverlapCache()--    4096MB for tigs - error profile overlaps.
OverlapCache()--       0MB for other processes.
OverlapCache()-- ---------
OverlapCache()--    4641MB for data structures (sum of above).
OverlapCache()-- ---------
OverlapCache()--      45MB for overlap store structure.
OverlapCache()--   11696MB for overlap data.
OverlapCache()-- ---------
OverlapCache()--   16384MB allowed.
OverlapCache()--
OverlapCache()-- Retain at least 79 overlaps/read, based on 39.90x coverage.
OverlapCache()-- Initial guess at 321 overlaps/read.
OverlapCache()--
OverlapCache()-- Adjusting for sparse overlaps.
OverlapCache()--
OverlapCache()--               reads loading olaps          olaps               memory
OverlapCache()--   olaps/read       all      some          loaded                 free
OverlapCache()--   ----------   -------   -------     ----------- -------     --------
OverlapCache()--          321   2384854       463        10306395  97.34%      11539 MB
OverlapCache()--      1633665   2385317         0        10588478 100.00%      11534 MB
OverlapCache()--
OverlapCache()-- Loading overlaps.
OverlapCache()--
OverlapCache()--          read from store           saved in cache
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--         736064 (006.95%)         707317 (006.68%)
OverlapCache()--        1035000 (009.77%)         999054 (009.44%)
OverlapCache()--        1655486 (015.63%)        1600005 (015.11%)
OverlapCache()--        2216982 (020.94%)        2148570 (020.29%)
OverlapCache()--        2813290 (026.57%)        2727998 (025.76%)
OverlapCache()--        3374147 (031.87%)        3270586 (030.89%)
OverlapCache()--        3918645 (037.01%)        3802097 (035.91%)
OverlapCache()--        4424776 (041.79%)        4294214 (040.56%)
OverlapCache()--        4961476 (046.86%)        4819175 (045.51%)
OverlapCache()--        5434907 (051.33%)        5281782 (049.88%)
OverlapCache()--        5828446 (055.05%)        5658247 (053.44%)
OverlapCache()--        6229626 (058.83%)        6049970 (057.14%)
OverlapCache()--        6614417 (062.47%)        6426187 (060.69%)
OverlapCache()--        7015289 (066.25%)        6816310 (064.37%)
OverlapCache()--        7410863 (069.99%)        7201174 (068.01%)
OverlapCache()--        7761929 (073.31%)        7546436 (071.27%)
OverlapCache()--        8127380 (076.76%)        7902291 (074.63%)
OverlapCache()--        8605250 (081.27%)        8360662 (078.96%)
OverlapCache()--        9170893 (086.61%)        8909065 (084.14%)
OverlapCache()--        9561111 (090.30%)        9288178 (087.72%)
OverlapCache()--        9861092 (093.13%)        9581187 (090.49%)
OverlapCache()--       10141289 (095.78%)        9857931 (093.10%)
OverlapCache()--       10389641 (098.12%)       10103541 (095.42%)
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--       10588478 (100.00%)       10299942 (097.28%)
OverlapCache()--
OverlapCache()-- Ignored 288133 duplicate overlaps.
OverlapCache()--
OverlapCache()-- Symmetrizing overlaps.
OverlapCache()--   Finding missing twins.
OverlapCache()--   Found 426 missing twins in 10299942 overlaps, 416 are strong.
OverlapCache()--   Dropping weak non-twin overlaps; allocated 0 MB scratch space.
OverlapCache()--   Dropped 9 overlaps; scratch space released.
OverlapCache()--   Adding 417 missing twin overlaps.
OverlapCache()--   Finished.

BestOverlapGraph()-- allocating best edges (72MB)

BestOverlapGraph()-- finding initial best edges.

BestOverlapGraph()-- filtering suspicious reads.
BestOverlapGraph()-- marked 1972279 reads as suspicious.

BestOverlapGraph()-- filtering high error edges.

BestOverlapGraph()-- filtering reads with lopsided best edges.

BestOverlapGraph()-- filtering spur reads.
BestOverlapGraph()-- detected 110070 spur reads and 1985641 singleton reads.
BestOverlapGraph()-- detected 3292 zombie reads.

BestOverlapGraph()-- removing best edges for contained reads.

==> BUILDING GREEDY TIGS.

breakSingletonTigs()-- Removed 105678 singleton tigs; reads are now unplaced.
optimizePositions()-- Optimizing read positions for 2385318 reads in 117107 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
optimizePositions()--   Recomputing positions, iteration 1, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 400845 reads
optimizePositions()--     changed:   1984473 reads
optimizePositions()--   Recomputing positions, iteration 2, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 404841 reads
optimizePositions()--     changed:   1980477 reads
optimizePositions()--   Recomputing positions, iteration 3, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 407594 reads
optimizePositions()--     changed:   1977724 reads
optimizePositions()--   Recomputing positions, iteration 4, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409155 reads
optimizePositions()--     changed:   1976163 reads
optimizePositions()--   Recomputing positions, iteration 5, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409967 reads
optimizePositions()--     changed:   1975351 reads
optimizePositions()--   Expanding short reads with 16 threads.
optimizePositions()--   Updating positions.
optimizePositions()--   Finished.

==> PLACE CONTAINED READS.

computeErrorProfiles()-- Computing error profiles for 117107 tigs, with 16 threads.
computeErrorProfiles()-- Finished.

placeContains()-- placing 204867 contained and 2142601 unplaced reads, with 16 threads.
placeContains()-- Placed 89525 contained reads and 10270 unplaced reads.
placeContains()-- Failed to place 115342 contained reads (too high error suspected) and 2132331 unplaced reads (lack of overlaps suspected).
optimizePositions()-- Optimizing read positions for 2385318 reads in 117107 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
bogart: bogart/AS_BAT_OptimizePositions.C:231: void Unitig::optimize_initPlace(uint32, optPos*, optPos*, bool, std::set<unsigned int>&, bool): Assertion `cnt > 0' failed.

Failed with 'Aborted'; backtrace (libbacktrace):
utility/system-stackTrace.C::89 in _Z17AS_UTL_catchCrashiP9siginfo_tPv()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
bogart/AS_BAT_OptimizePositions.C::231 in _ZN6Unitig18optimize_initPlaceEjP6optPosS1_bRSt3setIjSt4lessIjESaIjEEb()
bogart/AS_BAT_OptimizePositions.C::490 in _ZN9TigVector17optimizePositionsEPKcS1_._omp_fn.0()
../../../libgomp/team.c::120 in gomp_thread_start()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)() 
```
And I changed the -M to 16/40/100, the reported errors are the same.
But when I changed the -M to 256, I got the following error message:
```
==> PARAMETERS.

Resources:
  Memory                256 GB
  Compute Threads       16 (command line)

Lengths:
  Minimum read          0 bases
  Minimum overlap       500 bases

Overlap Error Rates:
  Graph                 0.144 (14.400%)
  Max                   0.144 (14.400%)

Deviations:
  Graph                 6.000
  Bubble                6.000
  Repeat                3.000

Edge Confusion:
  Absolute              2100
  Percent               200.0000

Unitig Construction:
  Minimum intersection  500 bases
  Maxiumum placements   2 positions

Debugging Enabled:
  (none)

==> LOADING AND FILTERING OVERLAPS.

ReadInfo()-- Using 2385317 reads, no minimum read length used.

OverlapCache()-- limited to 262144MB memory (user supplied).

OverlapCache()--      18MB for read data.
OverlapCache()--      90MB for best edges.
OverlapCache()--     236MB for tigs.
OverlapCache()--      63MB for tigs - read layouts.
OverlapCache()--      90MB for tigs - error profiles.
OverlapCache()--   65536MB for tigs - error profile overlaps.
OverlapCache()--       0MB for other processes.
OverlapCache()-- ---------
OverlapCache()--   66081MB for data structures (sum of above).
OverlapCache()-- ---------
OverlapCache()--      45MB for overlap store structure.
OverlapCache()--  196016MB for overlap data.
OverlapCache()-- ---------
OverlapCache()--  262144MB allowed.
OverlapCache()--
OverlapCache()-- Retain at least 79 overlaps/read, based on 39.90x coverage.
OverlapCache()-- Initial guess at 5385 overlaps/read.
OverlapCache()--
OverlapCache()-- Adjusting for sparse overlaps.
OverlapCache()--
OverlapCache()--               reads loading olaps          olaps               memory
OverlapCache()--   olaps/read       all      some          loaded                 free
OverlapCache()--   ----------   -------   -------     ----------- -------     --------
OverlapCache()--         5385   2385317         0        10588478 100.00%     195854 MB
OverlapCache()--
OverlapCache()-- Loading overlaps.
OverlapCache()--
OverlapCache()--          read from store           saved in cache
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--         736064 (006.95%)         707317 (006.68%)
OverlapCache()--        1035000 (009.77%)         999054 (009.44%)
OverlapCache()--        1655486 (015.63%)        1600005 (015.11%)
OverlapCache()--        2216982 (020.94%)        2148570 (020.29%)
OverlapCache()--        2813290 (026.57%)        2727998 (025.76%)
OverlapCache()--        3374147 (031.87%)        3270586 (030.89%)
OverlapCache()--        3918645 (037.01%)        3802097 (035.91%)
OverlapCache()--        4424776 (041.79%)        4294214 (040.56%)
OverlapCache()--        4961476 (046.86%)        4819175 (045.51%)
OverlapCache()--        5434907 (051.33%)        5281782 (049.88%)
OverlapCache()--        5828446 (055.05%)        5658247 (053.44%)
OverlapCache()--        6229626 (058.83%)        6049970 (057.14%)
OverlapCache()--        6614417 (062.47%)        6426187 (060.69%)
OverlapCache()--        7015289 (066.25%)        6816310 (064.37%)
OverlapCache()--        7410863 (069.99%)        7201174 (068.01%)
OverlapCache()--        7761929 (073.31%)        7546436 (071.27%)
OverlapCache()--        8127380 (076.76%)        7902291 (074.63%)
OverlapCache()--        8605250 (081.27%)        8360662 (078.96%)
OverlapCache()--        9170893 (086.61%)        8909065 (084.14%)
OverlapCache()--        9561111 (090.30%)        9288178 (087.72%)
OverlapCache()--        9861092 (093.13%)        9581187 (090.49%)
OverlapCache()--       10141289 (095.78%)        9857931 (093.10%)
OverlapCache()--       10389641 (098.12%)       10103541 (095.42%)
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--       10588478 (100.00%)       10299942 (097.28%)
OverlapCache()--
OverlapCache()-- Ignored 288133 duplicate overlaps.
OverlapCache()--
OverlapCache()-- Symmetrizing overlaps.
OverlapCache()--   Finding missing twins.
OverlapCache()--   Found 426 missing twins in 10299942 overlaps, 416 are strong.
OverlapCache()--   Dropping weak non-twin overlaps; allocated 0 MB scratch space.
OverlapCache()--   Dropped 9 overlaps; scratch space released.
OverlapCache()--   Adding 417 missing twin overlaps.
OverlapCache()--   Finished.

BestOverlapGraph()-- allocating best edges (72MB)

BestOverlapGraph()-- finding initial best edges.

BestOverlapGraph()-- filtering suspicious reads.
BestOverlapGraph()-- marked 1972279 reads as suspicious.

BestOverlapGraph()-- filtering high error edges.

BestOverlapGraph()-- filtering reads with lopsided best edges.

BestOverlapGraph()-- filtering spur reads.
BestOverlapGraph()-- detected 110070 spur reads and 1985641 singleton reads.
BestOverlapGraph()-- detected 3292 zombie reads.

BestOverlapGraph()-- removing best edges for contained reads.

==> BUILDING GREEDY TIGS.

breakSingletonTigs()-- Removed 105678 singleton tigs; reads are now unplaced.
optimizePositions()-- Optimizing read positions for 2385318 reads in 117107 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
optimizePositions()--   Recomputing positions, iteration 1, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 400845 reads
optimizePositions()--     changed:   1984473 reads
optimizePositions()--   Recomputing positions, iteration 2, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 404841 reads
optimizePositions()--     changed:   1980477 reads
optimizePositions()--   Recomputing positions, iteration 3, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 407594 reads
optimizePositions()--     changed:   1977724 reads
optimizePositions()--   Recomputing positions, iteration 4, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409155 reads
optimizePositions()--     changed:   1976163 reads
optimizePositions()--   Recomputing positions, iteration 5, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409967 reads
optimizePositions()--     changed:   1975351 reads
optimizePositions()--   Expanding short reads with 16 threads.
optimizePositions()--   Updating positions.
optimizePositions()--   Finished.

==> PLACE CONTAINED READS.

computeErrorProfiles()-- Computing error profiles for 117107 tigs, with 16 threads.
computeErrorProfiles()-- Finished.

placeContains()-- placing 204867 contained and 2142601 unplaced reads, with 16 threads.
placeContains()-- Placed 89525 contained reads and 10270 unplaced reads.
placeContains()-- Failed to place 115342 contained reads (too high error suspected) and 2132331 unplaced reads (lack of overlaps suspected).
optimizePositions()-- Optimizing read positions for 2385318 reads in 117107 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
bogart: bogart/AS_BAT_OptimizePositions.C:231: void Unitig::optimize_initPlace(uint32, optPos*, optPos*, bool, std::set<unsigned int>&, bool): Assertion `cnt > 0' failed.

Failed with 'Aborted'; backtrace (libbacktrace):

Failed with 'Bus error'; backtrace (libbacktrace):
```
How can I solve this?

**Comment by brianwalenz** [comment]
There's a fix for this in issue #1284.  Write back if you need help applying it.

**Comment by SixPlusSeven** [comment]
I have tried all the methods in issue #1284:

- Decrease memory;

- Change -dg;

- Change -db;

- updata AS_BAT_MergeOrphans.C;

But I get the same error logs:
```
==> PARAMETERS.

Resources:
  Memory                40 GB
  Compute Threads       16 (command line)

Lengths:
  Minimum read          0 bases
  Minimum overlap       500 bases

Overlap Error Rates:
  Graph                 0.144 (14.400%)
  Max                   0.144 (14.400%)

Deviations:
  Graph                 4.000
  Bubble                4.000
  Repeat                3.000

Edge Confusion:
  Absolute              2100
  Percent               200.0000

Unitig Construction:
  Minimum intersection  500 bases
  Maxiumum placements   2 positions

Debugging Enabled:
  (none)

==> LOADING AND FILTERING OVERLAPS.

ReadInfo()-- Using 2385317 reads, no minimum read length used.

OverlapCache()-- limited to 40960MB memory (user supplied).

OverlapCache()--      18MB for read data.
OverlapCache()--      90MB for best edges.
OverlapCache()--     236MB for tigs.
OverlapCache()--      63MB for tigs - read layouts.
OverlapCache()--      90MB for tigs - error profiles.
OverlapCache()--   10240MB for tigs - error profile overlaps.
OverlapCache()--       0MB for other processes.
OverlapCache()-- ---------
OverlapCache()--   10785MB for data structures (sum of above).
OverlapCache()-- ---------
OverlapCache()--      45MB for overlap store structure.
OverlapCache()--   30128MB for overlap data.
OverlapCache()-- ---------
OverlapCache()--   40960MB allowed.
OverlapCache()--
OverlapCache()-- Retain at least 79 overlaps/read, based on 39.90x coverage.
OverlapCache()-- Initial guess at 827 overlaps/read.
OverlapCache()--
OverlapCache()-- Adjusting for sparse overlaps.
OverlapCache()--
OverlapCache()--               reads loading olaps          olaps               memory
OverlapCache()--   olaps/read       all      some          loaded                 free
OverlapCache()--   ----------   -------   -------     ----------- -------     --------
OverlapCache()--          827   2385076       241        10480527  98.98%      29968 MB
OverlapCache()--      8150304   2385317         0        10588478 100.00%      29966 MB
OverlapCache()--
OverlapCache()-- Loading overlaps.
OverlapCache()--
OverlapCache()--          read from store           saved in cache
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--         736064 (006.95%)         707317 (006.68%)
OverlapCache()--        1035000 (009.77%)         999054 (009.44%)
OverlapCache()--        1655486 (015.63%)        1600005 (015.11%)
OverlapCache()--        2216982 (020.94%)        2148570 (020.29%)
OverlapCache()--        2813290 (026.57%)        2727998 (025.76%)
OverlapCache()--        3374147 (031.87%)        3270586 (030.89%)
OverlapCache()--        3918645 (037.01%)        3802097 (035.91%)
OverlapCache()--        4424776 (041.79%)        4294214 (040.56%)
OverlapCache()--        4961476 (046.86%)        4819175 (045.51%)
OverlapCache()--        5434907 (051.33%)        5281782 (049.88%)
OverlapCache()--        5828446 (055.05%)        5658247 (053.44%)
OverlapCache()--        6229626 (058.83%)        6049970 (057.14%)
OverlapCache()--        6614417 (062.47%)        6426187 (060.69%)
OverlapCache()--        7015289 (066.25%)        6816310 (064.37%)
OverlapCache()--        7410863 (069.99%)        7201174 (068.01%)
OverlapCache()--        7761929 (073.31%)        7546436 (071.27%)
OverlapCache()--        8127380 (076.76%)        7902291 (074.63%)
OverlapCache()--        8605250 (081.27%)        8360662 (078.96%)
OverlapCache()--        9170893 (086.61%)        8909065 (084.14%)
OverlapCache()--        9561111 (090.30%)        9288178 (087.72%)
OverlapCache()--        9861092 (093.13%)        9581187 (090.49%)
OverlapCache()--       10141289 (095.78%)        9857931 (093.10%)
OverlapCache()--       10389641 (098.12%)       10103541 (095.42%)
OverlapCache()--   ------------ ---------   ------------ ---------
OverlapCache()--       10588478 (100.00%)       10299942 (097.28%)
OverlapCache()--
OverlapCache()-- Ignored 288133 duplicate overlaps.
OverlapCache()--
OverlapCache()-- Symmetrizing overlaps.
OverlapCache()--   Finding missing twins.
OverlapCache()--   Found 426 missing twins in 10299942 overlaps, 416 are strong.
OverlapCache()--   Dropping weak non-twin overlaps; allocated 0 MB scratch space.
OverlapCache()--   Dropped 9 overlaps; scratch space released.
OverlapCache()--   Adding 417 missing twin overlaps.
OverlapCache()--   Finished.

BestOverlapGraph()-- allocating best edges (72MB)

BestOverlapGraph()-- finding initial best edges.

BestOverlapGraph()-- filtering suspicious reads.
BestOverlapGraph()-- marked 1972279 reads as suspicious.

BestOverlapGraph()-- filtering high error edges.

BestOverlapGraph()-- filtering reads with lopsided best edges.

BestOverlapGraph()-- filtering spur reads.
BestOverlapGraph()-- detected 110076 spur reads and 1986904 singleton reads.
BestOverlapGraph()-- detected 3292 zombie reads.

BestOverlapGraph()-- removing best edges for contained reads.

==> BUILDING GREEDY TIGS.

breakSingletonTigs()-- Removed 106686 singleton tigs; reads are now unplaced.
optimizePositions()-- Optimizing read positions for 2385318 reads in 118195 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
optimizePositions()--   Recomputing positions, iteration 1, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 400756 reads
optimizePositions()--     changed:   1984562 reads
optimizePositions()--   Recomputing positions, iteration 2, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 404846 reads
optimizePositions()--     changed:   1980472 reads
optimizePositions()--   Recomputing positions, iteration 3, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 407567 reads
optimizePositions()--     changed:   1977751 reads
optimizePositions()--   Recomputing positions, iteration 4, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409152 reads
optimizePositions()--     changed:   1976166 reads
optimizePositions()--   Recomputing positions, iteration 5, with 16 threads.
optimizePositions()--     Reset zero.
optimizePositions()--     Checking convergence.
optimizePositions()--     converged: 409964 reads
optimizePositions()--     changed:   1975354 reads
optimizePositions()--   Expanding short reads with 16 threads.
optimizePositions()--   Updating positions.
optimizePositions()--   Finished.

==> PLACE CONTAINED READS.

computeErrorProfiles()-- Computing error profiles for 118195 tigs, with 16 threads.
computeErrorProfiles()-- Finished.

placeContains()-- placing 204044 contained and 2143177 unplaced reads, with 16 threads.
placeContains()-- Placed 89171 contained reads and 10381 unplaced reads.
placeContains()-- Failed to place 114873 contained reads (too high error suspected) and 2132796 unplaced reads (lack of overlaps suspected).
optimizePositions()-- Optimizing read positions for 2385318 reads in 118195 tigs, with 16 threads.
optimizePositions()--   Allocating scratch space for 2385318 reads (149082 KB).
optimizePositions()--   Initializing positions with 16 threads.
bogart: bogart/AS_BAT_OptimizePositions.C:231: void Unitig::optimize_initPlace(uint32, optPos*, optPos*, bool, std::set<unsigned int>&, bool): Assertion `cnt > 0' failed.

Failed with 'Aborted'; backtrace (libbacktrace):
utility/system-stackTrace.C::89 in _Z17AS_UTL_catchCrashiP9siginfo_tPv()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
bogart/AS_BAT_OptimizePositions.C::231 in _ZN6Unitig18optimize_initPlaceEjP6optPosS1_bRSt3setIjSt4lessIjESaIjEEb()
bogart/AS_BAT_OptimizePositions.C::490 in _ZN9TigVector17optimizePositionsEPKcS1_._omp_fn.0()
../../../libgomp/team.c::120 in gomp_thread_start()
(null)::0 in (null)()
(null)::0 in (null)()
(null)::0 in (null)()
```

**Comment by brianwalenz** [comment]
Well, crud.  Can you share the data so I can properly debug?

**Comment by SixPlusSeven** [comment]
Sorry, I can't provide this data because of some confidentiality agreements...
Thank you for your help!

**Comment by skoren** [comment]
Should be fixed by #1414. If you have a chance to test, give that a shot.
